cupyx/scipy/fftpack/fft.py

```diff
@@ -70,6 +104,13 @@ def fft(x, n=None, axis=-1, overwrite_x=False):
             ``axis`` is used.
         axis (int): Axis over which to compute the FFT.
         overwrite_x (bool): If True, the contents of ``x`` can be destroyed.
+        plan (cupy.cuda.cufft.Plan1d): a cuFFT plan for transforming ``x``
+            over ``axis``, which can be obtained using
```

Comment:
I think this needs to have a : at the end of the line for the indentation of the nxt line to render correctly, but I haven't built the docs to check

Reply:
Oh nice, I didn't know this subtlety, thanks! I looked up an example I found [here](https://www.sphinx-doc.org/en/1.7/ext/example_google.html) and probably we need `::`?

Reply:
Done btw.